meson.build

```diff
@@ -340,7 +340,7 @@ elif (get_option('crypto') == 'gnutls')
 elif (get_option('crypto') == 'openssl')
   if (get_option('openssl_dir') != '')
     crypto = declare_dependency(
-      link_args: ['-L' + get_option('openssl_dir'), '-llibssl_static', '-llibcrypto_static'],
+      link_args: ['-L' + get_option('openssl_dir'), '-L' + get_option('openssl_dir') + '/lib', '-llibssl_static', '-llibcrypto_static'],
```

Comment:
I think it should be `get_option('openssl_dir') / 'lib'`, but...

Reply:
It feels like the CI has a version of meson that doesn't support the syntax.

Reply:
Windows CI version is always the updated from PyPI (0.54.3), so it should support. Didn't check Travis, tho